src/parser/parser.yy

```diff
@@ -1483,6 +1484,12 @@ set_config_sentence
     }
     ;
 
+balance_sentence
+    : KW_BALANCE KW_LEADER {
+        $$ = new BalanceSentence(BalanceSentence::SubType::kLeader);
```

Comment:
Could remove this type parameter ?

Reply:
There may be other subtype later, for example `balance data`.

Reply:
Maybe add a comment will be better？